### PR TITLE
Update application

### PR DESCRIPTION
**Pull request by gleich**
1. Added the use of go modules
2. Unexport exported variables and functions

I just thought that these changes should be implemented before the cask is released.

### CODE REVIEW

File: frc.go [hunk 1]
```diff
@@ -142,21 +142,21 @@ func ListInfo(header string, titles []string, data []interface{}) {
...
+func displayTeam(team tbago.Team) {
 	header := fmt.Sprintf("Team %d", team.TeamNumber)
 	titles := []string{"Nickname", "Website", "Rookie", "Country", "Motto"}
 	data := []interface{}{team.Nickname, team.Website, team.RookieYear, team.Country, team.Motto}
-	ListInfo(header, titles, data)
+	listInfo(header, titles, data)
 }
 
-func DisplayEvent(event tbago.Event) {
+func displayEvent(event tbago.Event) {
```
Comment (ErikBoesen) [inline frc.go:152]: Is this sort of camelCase conventional for unexported functions?

**Review by ErikBoesen — APPROVED** [review]

File: frc.go (hunk 1 above)
Comment (gleich) [reply frc.go:152]: Yep! When a function name starts with an uppercase character it is exported

Comment (gleich) [reply frc.go:152]: https://tour.golang.org/basics/3

**Comment by gleich** [comment]
Just so ya know I can't merge even if you approve because I am an outside contributor. Thanks for approving! Glad to contribute

**Review by ErikBoesen — APPROVED** [review]

**Pull request merged**